certbot/CHANGELOG.md

```diff
@@ -29,8 +29,12 @@ More details about these changes can be found on our GitHub repo.
 
 ### Added
 
-* Added the ability to remove email and phone contact information from an account 
+* Added the ability to remove email and phone contact information from an account
```

Comment:
Could you move the new changelog entries to the 1.9.0 section?

Reply:
Done :)

Reply:
I had nothing to do with the email ability, it's just my editor deleting trailing whitespace.